hugophotoswipe/photo.py

```diff
@@ -70,11 +74,16 @@ def __init__(
         # names
         self.name = name
         self.alt = alt
-        self.caption = caption
+        self._caption = caption
 
         # other
-        self.copyright = copyright
+        self._copyright = copyright
         self.cover_path = None
+        self._exif = None
+        self._iptc = None
+
+        # process load image to ensure it's a valid image.
+        self.original_image
```

Comment:
This breaks the resiliency for deleted photos you had in the current master. This now results in a FileNotFoundError that isn't caught in the Photo() call of _load_albums. See also https://github.com/sebastiaan-lampo/HugoPhotoSwipe/issues/3

More worrying to me is that this is called for every photo in every album on every action unless a specific album is specified. That could severely impact performance. 

I may have been too eager with this feature and can't recommend merging this PR into master anymore. Would love to get your feedback though.